demos/imagenet_classification_demo/grid_mat.hpp

```diff
@@ -0,0 +1,176 @@
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.width))
+                                    ),
+                          std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.width)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.height))
+                                    )},
+                     currSourceID{0} {
+        int minCellSize = std::min(maxDisp.width / size.width, maxDisp.height / size.height);
+        cellSize = cv::Size(minCellSize, minCellSize);
+        rectangleHeight = presenter.graphSize.height;
+
+        for (int i = 0; i < size.width; i++) {
+            for (int j = 0; j < size.height; j++) {
+                cv::Point p;
+                p.x = cellSize.width * i;
+                p.y = rectangleHeight + (cellSize.height * j);
+                points.push_back(p);
+            }
+        }
+
+        outImg.create((cellSize.height * size.height) + rectangleHeight, cellSize.width * size.width, CV_8UC3);
+        outImg.setTo(0);
+    }
+
+    cv::Size getSize() {
+        return size;
+    }
+
+    void listUpdate(std::list<std::tuple<cv::Mat, std::string, bool>>& imageInfos) {
+        if (!imageInfos.empty()) {
+            updateList.splice(updateList.end(), imageInfos);
+        }
+    }
+
+    void textUpdate(double avgFPS, double avgLatency, double accuracy,
+                    bool isFpsTest, bool showAccuracy,
+                    Presenter& presenter) {
+        // Draw a rectangle
+        cv::Point p1 = cv::Point(0, 0);
+        cv::Point p2 = cv::Point(outImg.cols, rectangleHeight);
+        rectangle(outImg, p1, p2, cv::Scalar(0, 0, 0), cv::FILLED);
+
+        presenter.drawGraphs(outImg);
+        
+        // Show FPS
+        double fontScale = 1.5;
+        int thickness = 2;
+        int baseline = 0;
+        cv::Size textSize = cv::getTextSize("", cv::FONT_HERSHEY_PLAIN, fontScale, thickness, &baseline);
+        cv::Scalar textColor = cv::Scalar(255, 255, 255);
+        int textPadding = 10;
+
+        cv::putText(outImg,
+                    cv::format("FPS: %0.01f", avgFPS),
+                    cv::Point(textPadding, textSize.height + textPadding),
+                    cv::FONT_HERSHEY_PLAIN, fontScale, textColor, thickness);
+        cv::putText(outImg,
+                    cv::format("Latency: %dms", static_cast<int>(avgLatency * 1000)),
+                    cv::Point(textPadding, (textSize.height + textPadding) * 2),
+                    cv::FONT_HERSHEY_PLAIN, fontScale, textColor, thickness);
+        
+        if (showAccuracy) {
+            std::string accuracyMessage = "Accuracy (top 0): 0.000";
+            cv::Size accuracyMessageSize = cv::getTextSize(accuracyMessage,
```

Comment:
That size can be stored in the constructor as a field to avoid computing it each time.